integration_tests/suite/test_setup.py

```diff
@@ -477,7 +477,7 @@ def test_setup_valid_stops(self):
         def setupd_is_stopped():
             return not self.service_status('setupd')['State']['Running']
 
-        until.true(setupd_is_stopped, timeout=10)
+        until.true(setupd_is_stopped, timeout=15)
```

Comment:
Avoid solving issue with only increasing timeout without reason
Please make this modification into a separate commit and add justification why you really need to increase to timeout

Reply:
I don't know why. I didn't change anything, but it fails systematically with previous timeout. No errors or warnings.

Reply:
@sduthil fixed it in wazo-test-helpers. It's a problem with docker-compose 2+

Reply:
The fix: https://github.com/wazo-platform/wazo-test-helpers/pull/82